### PR TITLE
[bitnami/tomcat] feat: allow custom values for labels and matchLabels

### LINKED ISSUE #4314: [bitnami/tomcat] Allow custom common labels
**Which chart**:
tomcat 7.0.0

**Is your feature request related to a problem? Please describe.**
No option to add custom labels. 

**Describe the solution you'd like**
Add commonLabels value to allow custom labels to be passed to tomcat. A couple charts already do this, such as https://github.com/bitnami/charts/tree/master/bitnami/airflow

Not sure if this works with the helper function to define "tomcat.labels", airflow chart seems to avoid using their method

### PR DESCRIPTION
**Pull request by michaelbonadio**
**Description of the change**

add values for labels and matchLabels to allow additional, custom labels

**Benefits**

Allows additional, custom labels and match labels

**Possible drawbacks**

**Applicable issues**

  - fixes #4314

**Additional information**

~~I also changed the helper templates to use the common labels/ match labels instead of hard coding them.~~ 
Edit: Linting fails using common labels template, reverted back to hard coded values

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/tomcat/values.yaml [hunk 1]
```diff
@@ -269,3 +269,11 @@ readinessProbe:
   periodSeconds: 51
   failureThreshold: 3
   timeoutSeconds: 3
+
+## Additional labels to apply to all resources
+labels: {}
```
Comment (andresbono) [inline bitnami/tomcat/values.yaml:274]: Can you use `commonLabels` instead? For standardization with other charts, i.e: `bitnami/discourse`.

File: bitnami/tomcat/values.yaml [hunk 2]
```diff
@@ -269,3 +269,11 @@ readinessProbe:
...
+#   foo: bar
+
+## Additional labels to use on deploy.spec.selector.matchLabels and svc.spec.selector
+matchLabels: {}
```
Comment (andresbono) [inline bitnami/tomcat/values.yaml:278]: Why would it be useful to modify the current matchLabels with extra ones?

**Review by andresbono — COMMENTED** [review]
Thanks for your contribution! Could you please check my comments?

File: bitnami/tomcat/values.yaml (hunk 2 above)
Comment (michaelbonadio) [reply bitnami/tomcat/values.yaml:278]: I am not sure. At the time I thought I saw other charts did this so I figured might as well while Im here, but looking now I can't find any so I'll remove this

**Comment by michaelbonadio** [comment]
@andresbono can you re-review now that I addressed your comments?

**Review by marcosbc — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**